src/natcap/invest/urban_nature_access.py

```diff
@@ -97,19 +100,34 @@
                 "pixel."
             ),
         },
-        'admin_unit_vector_path': {
+        'aoi_vector_path': {
             'type': 'vector',
             'name': 'administrative boundaries',
             'geometries': spec_utils.POLYGONS,
-            'fields': {},  # TODO, complete required fields (if any)
-            'about': "",  # TODO, will know more about this when I implement.
+            'fields': {
+                "pop_[POP_GROUP]": {
+                    "type": "number",
+                    "units": u.none,
+                    "required": False,
+                    "about": gettext(
+                        "The proportion (numeric values in the range 0-1) of "
+                        "the population within this region belonging to the "
+                        "identified population group."
+                    ),
+                }
+            },
+            'about': gettext(
+                "Non-overlapping regions (typically administrative "
```

Comment:
Out of curiosity, which operation requires that the polygons are non-overlapping?

Reply:
And is that something that's worth validating in `validate`?

Reply:
Currently, the reason for this is that we rasterize a population group field all at once in [`_rasterize_aois`](https://github.com/natcap/invest/pull/1024/files#diff-3c80dfa184fdd56cb7d4dba74d5215d5f1f4ab52284cd104e1dff04d8035a952R572), so each pixel can only belong to a single polygon.

But the truth is that this is really the only function in the whole model that actually relies on geometries being non-overlapping, and there's no real reason why this should have to be the case ... supporting overlapping admin units would just be a little more looping.  So, I'll work this in.

And then that will remove the need to validate that constraint.

- [ ] Refactor to support overlapping admin units
- [ ] Remove this note from the `ARGS_SPEC`

Reply:
OK @emlys !  I've refactored this in bca7408db, but the outputs definitely do not look particularly good since we're using pygeoprocessing's definition of disjoint, where polygons can't even be touching (see below).  So I'm going to ask what the science team would like to do about this, and if they want something specific I can PR that in separately.

In any case, this slight refactor is ready for review!

<img width="789" alt="Screen Shot 2022-07-18 at 10 15 47 PM" src="https://user-images.githubusercontent.com/4583897/179671052-8f3db016-2326-47c2-a265-eba0543a0ac1.png">

Reply:
OK I was dissatisfied with my solution, because while it's the best general-case solution, it leaves users with AOIs that touch but do not overlap in the lurch with outputs that are harder to interpret.  667e1b4d7 does a little more work to determine whether we need to do the fracturing above.  If we do, then we do and you'll get the disjoint sets above.  If not, then the whole admin units vector is treated as a single layer, as below:
<img width="831" alt="Screen Shot 2022-07-18 at 10 57 23 PM" src="https://user-images.githubusercontent.com/4583897/179676479-d5d4452a-85d5-490f-b19c-0405318c146a.png">

Reply:
OK!  So I'm just getting back to this and this really turned into an interesting conversation!

After discussing tradeoffs with the science team, they decided that we should not go through the extra effort of supporting possibly-overlapping polygons.  Rev 53af0c709 reverts the state of the codebase to before I started trying to make the above work reasonably.  In case the user provides a vector with admin units that overlap (greater than a reasonable numerical tolerance), I'll add a function that logs a warning.

That being said, it seems like a useful thing for cases like this, where we may not actually have disjoint polygons that can be rasterized in one pass, so we'll handle this in https://github.com/natcap/pygeoprocessing/issues/269.